src/actions/actions/PaymentMethod/WyreAccount.js

```diff
@@ -234,4 +236,19 @@ export const sendTransaction = (_, fromCurr, fromVal, toCurr, navigation) => asy
     }
 };
 
+export const getExchangeRates = () => async(dispatch) => {
+    dispatch(getActiveTransaction());
+    try {
+        const { data }= await WyreService.build().getRates();
+        if ( data) {
+            dispatch(getActiveTransactionResponse(data));
+        } else {
+            DelayedAlert('Failed fetching Exchange rates');
+        }
+    } catch (error) {
+        dispatch(getActiveTransactionResponse());
```

Comment:
we should create a separate action for handling errors for example getActiveTransactionError